packages/grid/package.json

```diff
@@ -35,6 +37,5 @@
   ],
   "publishConfig": {
     "access": "public"
-  },
-  "style": "src/index.css"
```

Comment:
@austin94 why did you remove this? It informs `postcss-import` where to load from if someone decides to use `@import '@zendeskgarden/css-grid'`.

Reply:
@ryanseddon did some research and mentioned that postcss-import will default to `main` if there is no `style` provided. Also, that file doesn't exist in `src/`

Reply:
Oh right. Unlike the other packages, this is Sass source code. So no `postcss-import`.